cmd/nerdctl/load.go

```diff
@@ -68,6 +72,16 @@ func loadAction(cmd *cobra.Command, args []string) error {
 		in = f
 	}
 
+	inbuf := bufio.NewReader(in)
+	buff, _ := inbuf.Peek(3)
+	if bytes.HasPrefix(buff, []byte("\x1f\x8b")) {
+		in, _ = gzip.NewReader(inbuf)
+	} else if bytes.HasPrefix(buff, []byte("\x42\x5a\x68")) {
+		in = bzip2.NewReader(inbuf)
+	} else {
+		in = inbuf
+	}
```

Comment:
Thanks for working on this, but I'd suggest using https://pkg.go.dev/github.com/containerd/containerd/archive/compression#DecompressStream

Reply:
`DecompressStream` only supports gzip and zstd.
https://github.com/containerd/containerd/blob/main/archive/compression/compression.go#L169

but `docker load` support gzip and bzip2.
https://docs.docker.com/engine/reference/commandline/load/#extended-description

Reply:
Then we should fix containerd/containerd

Reply:
Anyway, supporting bzip2 is probably not important. Is anyone really using bzip2 for Docker v1 archives?

Reply:
@AkihiroSuda I first change to  using`DecompressStream`,